test_mcp.py

```
@@ -3,9 +3,53 @@
+            os.environ["PFSENSE_CONNECTION_METHOD"] = "ssh"
+            # Ensure necessary SSH config env vars are set to avoid other errors
+            # if the manager tried to connect before our intended check.
+            os.environ["PFSENSE_SSH_HOST"] = "dummy_host"
+            os.environ["PFSENSE_SSH_USERNAME"] = "dummy_user"
+            os.environ["PFSENSE_SSH_PASSWORD"] = "dummy_pass" # For non-key auth path
+
+            manager = PfSenseConnectionManager()
+
+            try:
+                await manager.execute("this_is_an_invalid_command_exploit")
+                assert False, "ValueError was not raised for invalid SSH command"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)